tests/datetime/test_datetime.py

```diff
@@ -44,16 +44,15 @@ def test_can_find_before_the_year_2000():
 
 @checks_deprecated_behaviour
 def test_can_find_each_month():
-    for i in hrange(1, 12):
-        minimal(datetimes(), lambda x: x.month == i)
+    for i in hrange(12):
+        datetimes().filter(lambda x: x.month == i + 1).example()
```

Comment:
Then it breaks when running under nose - I'd also like to parametrize them, but can't see a way that's worth the trouble.